.evergreen/install-dependencies.sh

```diff
@@ -19,42 +19,42 @@ mkdir -p ${NODE_ARTIFACTS_PATH}
 mkdir -p ${NPM_CACHE_DIR}
 mkdir -p "${NPM_TMP_DIR}"
 
-case $NODE_LTS_NAME in
-  "argon")
-    VERSION=4
-    ;;
-  "boron")
-    VERSION=6
-    ;;
-  "carbon")
-    VERSION=8
-    ;;
-  "dubnium")
-    VERSION=10
-    ;;
-  "erbium")
-    VERSION=12
-    ;;
-  "fermium")
-    VERSION=14
-    ;;
-  "gallium")
-    VERSION=16
-    ;;
-  "hydrogen")
-    VERSION=18
-    ;;
-  "iron")
-    VERSION=20
-    ;;
-  *)
-    echo "Unsupported Node LTS version $1"
-    exit 1
-    ;;
-esac
-
-NODE_VERSION=$(curl --retry 8 --retry-delay 5  --max-time 50 --silent -o- https://nodejs.org/download/release/latest-v${VERSION}.x/SHASUMS256.txt | head -n 1 | awk '{print $2};' | cut -d- -f2)
-export NODE_VERSION=${NODE_VERSION:1} # :1 gets rid of the leading 'v'
+get_node_version() {
+  case $NODE_LTS_NAME in
```

Comment:
Since this still relies on global env vars what does the function syntax provide here? I don't disagree with the change, but perhaps we want to pass in `$NODE_LTS_NAME` and return `$NODE_VERSION`.